packages/atomic/cypress/utils/componentUtils.ts

```diff
@@ -1,14 +1,17 @@
-export async function getTextOfAllElements(selector: string) {
-  return new Promise((resolve) => {
-    cy.get(selector).then((elems) => {
-      const originalValues = [...elems].map((el: any) => el.textContent.trim());
-      resolve(originalValues);
-    });
-  });
-}
-
 export function doSortAlphanumeric(originalValues: string[]) {
   return originalValues
     .concat()
     .sort((first, second) => first.localeCompare(second));
 }
+
+export function revertFormatNumericFacet(num: string) {
+  return Number(num.replace(/,/g, ''));
+}
+
+export function revertFormatedDateFacet(date: string) {
```

Comment:
Names are a bit inconsistent: Formated here and Format for the previous method.

Reply:
Why not have those in facet utils?

Reply:
sure will do

Reply:
I updated the name, and moved them to FacetUtil